cmd/embedded-cluster/uninstall.go

```diff
@@ -404,6 +412,18 @@ var resetCommand = &cli.Command{
 			}
 		}
 
+		if c.Bool("reboot") {
+			if prompts.New().Confirm("Continue with reboot?", false) {
```

Comment:
I'm not sure we really need this, but it felt like the responsible thing to do.

Reply:
if you want to add a confirm here, you'll need to only do so when we don't pass "--no-prompts" IMO

Reply:
Given that someone passed the reboot flag, I don't think we need a confirmation for this.